### PR TITLE
docs: create docs folder and move README.md

### PR DESCRIPTION
**Pull request by stu01509**
According to this https://github.com/apache/apisix-website/issues/216, so I created the folder `/docs/en/latest` to save the current `README.md` file.

### CODE REVIEW

**Comment by stu01509** [comment]
Hi @juzhiyuan 

I created the PR, please take a look is correct? and Should I copy all of `xxx.md` files under the `/charts`? to `/docs/en/latest` ?

**Comment by juzhiyuan** [comment]
Hi, yes, you could follow this step:

1. **Move** all `.md` files under the `chart` folder to `/en/latest/xxx`;
2. No need to update README, just keep it under the root folder, for our website doesn't show README.

**Comment by stu01509** [comment]
Hi @juzhiyuan 

Update the commit :)

**Review by juzhiyuan — COMMENTED** [review]
LGTM from structure, and please add the `config.json` file under the `lastest` folder.

**Comment by stu01509** [comment]
Hi @juzhiyuan 

I following the https://github.com/apache/apisix-dashboard/blob/master/docs/en/latest/config.json to create the `config.json` file, please take a look.

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by juzhiyuan** [comment]
Once this PR gets merged, @stu01509 please update apisix-website's sync codes.

**Comment by stu01509** [comment]
> Once this PR gets merged, @stu01509 please update apisix-website's sync codes.

Sure.

File: docs/en/latest/config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "version": 0.4,
+    "sidebar": [
+      {
+        "type": "category",
+        "label": "Installation",
+        "items": ["apisix", "apisix-dashboard", "apisix-ingress-controller"]
+      }
+    ]
+  }
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:10]: new line

File: docs/en/latest/config.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "version": 0.4,
+    "sidebar": [
+      {
+        "type": "category",
+        "label": "Installation",
+        "items": ["apisix", "apisix-dashboard", "apisix-ingress-controller"]
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:7]: don't forget to add meta infomation.

File: docs/en/latest/config.json [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+      "type": "category",
+      "label": "Installation",
+      "items": [
+        "apisix",
+        "apisix-dashboard",
+        "apisix-ingress-controller"
+      ]
+    }
+  ]
+}
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:14]: image

Need a blank line at the ned of file

Comment (stu01509) [reply docs/en/latest/config.json]: Sorry, the auto prettier remove this, update the commit :)

**Review by tokers — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Pull request merged**